internal/pkg/telemetry/span.go

```diff
@@ -1,24 +1,10 @@
 package telemetry
 
 import (
-	"go.opentelemetry.io/otel/attribute"
 	"go.opentelemetry.io/otel/codes"
 	"go.opentelemetry.io/otel/trace"
-	ddext "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 )
 
-func SampleRate(v float64) attribute.KeyValue {
-	return attribute.Float64(ddext.EventSampleRate, v)
-}
-
-func KeepSpan() attribute.KeyValue {
-	return attribute.Bool(ddext.ManualKeep, true)
-}
-
-func DropSpan() attribute.KeyValue {
-	return attribute.Bool(ddext.ManualDrop, true)
-}
-
```

Comment:
Prvy commit, mali sme tu nejake funkcie pre nastavovanie DD atributov.
Uz sa v kode pouzivalo iba `KeepSpan`, co sme pouzivali, ked sme chceli indexovat nejaky non-root span, pre dashboard. To uz nepotrebujeme, ked mame `metrics`.